scripts/staging/validate.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# --- Default Variables ---
+CHART_VALIDATE="false"
+
+# --- Parse arguments early so CHART_VALIDATE is set before using it ---
```

Comment:
Why can't we parse the arguments first?

Reply:
Ok, let me move all others to the top of script similar to CHART_VALIDATE